installer/pkg/workflow/utils.go

```diff
@@ -23,9 +23,8 @@ const (
 	internalFileName = "internal.yaml"
 	joinWorkersStep  = "joining_workers"
 	mastersStep      = "masters"
-	newTLSStep       = "newtls"
+	newTLSStep       = "tls"
```

Comment:
This should also be `tlsStep`, not `newTLSStep`, now that there will no longer be a need to distinguish between two TLS approaches.